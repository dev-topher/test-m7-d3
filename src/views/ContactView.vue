<!-- src/views/ContactView.vue -->
<template>
    <div>
      <h1>Contact Us</h1>
      <p>This is the Contact page.</p>
    </div>
  </template>
  
  <script>
  export default {
    name: 'ContactView',
  };
  </script>
  
  <style scoped>
  /* Estilos espec√≠ficos para ContactView.vue */
  </style>
  