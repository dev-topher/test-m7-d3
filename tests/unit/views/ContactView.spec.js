// tests/unit/views/ContactView.spec.js
import { shallowMount } from '@vue/test-utils';
import ContactView from '@/views/ContactView.vue';

describe('ContactView.vue', () => {
  it('renders the Contact page', () => {
    const wrapper = shallowMount(ContactView);
    expect(wrapper.text()).toContain('Contact Us');
    expect(wrapper.text()).toContain('This is the Contact page.');
  });

  it('matches the snapshot', () => {
    const wrapper = shallowMount(ContactView);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
