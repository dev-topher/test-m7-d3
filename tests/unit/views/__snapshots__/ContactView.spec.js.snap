// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactView.vue matches the snapshot 1`] = `
<div>
  <h1>Contact Us</h1>
  <p>This is the Contact page.</p>
</div>
`;
